generate/generate.go

```diff
@@ -35,7 +35,7 @@ const msgpMode = gen.Encode | gen.Decode | gen.Marshal | gen.Unmarshal | gen.Siz
 // The package name is set to the outputPath directory name.
 //
 // NOTE: If oneOf/allOf entries exist than any JSON schema instances in the root schema file will be skipped.
-func BuildStructs(schemaPath string, outputDir string, useMessagePack bool) error {
+func BuildStructs(schemaPath string, outputDir string, useMessagePack bool, nameMap map[string]string) error {
```

Comment:
this should bump hte minor/major version or a backwards compatible method signature also needs to be implemented

Reply:
It is a good question what the expected API of this is, given it is packaged as a program.  Since this function is capitalized it could have potentially been used so I'll make it a wrapper function.